services/license.js

```diff
@@ -0,0 +1,31 @@
+const assert = require('assert');
+
+function validationError(attribute) {
+  return `License: Invalid "${attribute}" provided`;
+}
+
+function assertParams({ id, name, groups, compatibility, regexp, url }) {
```

Comment:
Minor of death, but `validateParams` would sound a tiny bit more appropriate to me here. Especially when looking at the name of the error that's raised when things aren't valid 💥